web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -543,35 +582,36 @@ div#main-content {
 #block-cj-material-local-actions nav .button.button-action,
 input.button {
   margin-top: 1em;
+  margin-left: 0;
   border: 3px solid transparent;
   border-radius: 2px;
   color: #FFFFFF;
   background: #1E5168;
   cursor: pointer;
   display: inline-block;
-  font-size: 16px;
+  font-size: 13px;
```

Comment:
AAAAAH! Prime numbers. If this hasn't been very specifically selected, lets go to 12px as that is a size used elsewhere or 14/16

Reply:
Done, and agreed - have amended this to 14px